lib/pages/farm/components/my_farm_item.dart

```diff
@@ -102,88 +101,57 @@ Widget myFarmItem(
+        SizedBox(
+          width: cardWidth,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(
+                'Currently Staked',
+                style: txStyle,
+              ),
+              Obx(
+                () => Text(
+                  '${parsedView.toStringAsFixed(4)} ${farm.strStakedSymbol}',
                   style: txStyle,
                 ),
-                Obx(
-                  () => Text(
-                    '''${double.parse(farm.stakedInfo.value.viewAmount).toStringAsFixed(4)} ${farm.strStakedSymbol}''',
-                    style: txStyle,
-                  ),
-                )
-              ],
-            ),
+              ),
+            ],
           ),
-          SizedBox(
-            width: cardWidth,
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                Text(
-                  'Rewards Earned',
+        ),
+        SizedBox(
+          width: cardWidth,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(
+                'Rewards Earned',
+                style: txStyle,
+              ),
+              Obx(
+                () => Text(
+                  '${farm.strRewards} ${farm.strRewardSymbol}',
                   style: txStyle,
                 ),
-                Obx(
-                  () => Text(
-                    '${farm.strRewards} ${farm.strRewardSymbol}',
-                    style: txStyle,
-                  ),
-                )
-              ],
-            ),
-          ),
-          SizedBox(
-            width: cardWidth,
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                Text('Total AX available (Staked + Earned)', style: txStyle),
-                Obx(
-                  () => Text(
-                    '''${(double.parse(farm.stakedInfo.value.viewAmount) + double.parse(farm.strRewards.value)).toStringAsFixed(2)} ${farm.strRewardSymbol}''',
-                    style: txStyle,
-                  ),
-                )
-              ],
-            ),
+              )
+            ],
           ),
-        ] else ...[
-          SizedBox(
-            width: cardWidth,
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                Text(
-                  'LP tokens provided',
+        ),
+        SizedBox(
+          width: cardWidth,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text('Total AX available (Staked + Earned)', style: txStyle),
+              Obx(
+                () => Text(
+                  '${parsedTotal.toStringAsFixed(2)} ${farm.strRewardSymbol}',
```

Comment:
When manipulating variables would be better to do it separately in a new variable, so that the interpolation is shorter and easier to read/understand.